client/common/IconButton.tsx

```diff
@@ -0,0 +1,31 @@
+import React, { ComponentType } from 'react';
+import styled from 'styled-components';
+import Button, { ButtonProps } from './Button';
+import { remSize } from '../theme';
+
+const ButtonWrapper = styled(Button)`
+  width: ${remSize(48)};
+  > svg {
+    width: 100%;
+    height: 100%;
+  }
+`;
+
+export type IconButtonProps = Omit<
+  ButtonProps,
+  'iconBefore' | 'display' | 'focusable'
+> & {
+  icon?: ComponentType<{ 'aria-label'?: string }> | null;
```

Comment:
same here maybe prefer to remove the | null and let the nullish cases all be undefined